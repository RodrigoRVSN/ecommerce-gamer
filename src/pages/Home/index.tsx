import { Button, Container } from "@material-ui/core";

import Grid from "@material-ui/core/Grid";

import { Products } from "../../components/Products";
import { OrderMenu } from "../../components/OrderMenu";

export function Home(): JSX.Element {
  return (
    <>
      <Container>
        <OrderMenu />
        <Grid
          spacing={4}
          container
          style={{
            alignItems: "center",
            justifyContent: "center",
            textAlign: "center",
          }}
        >
          <Products />
        </Grid>
      </Container>
    </>
  );
}
