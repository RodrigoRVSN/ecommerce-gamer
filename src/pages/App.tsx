/* eslint-disable no-use-before-define */
import React, { BrowserRouter, Route, Switch } from "react-router-dom";
import { Header } from "../components/Header";

import { Home } from "./Home";

function App(): JSX.Element {
  return (
    <>
      <BrowserRouter>
        <Header />
        <Switch>
          <Route path="/" exact component={Home} />
        </Switch>
      </BrowserRouter>
    </>
  );
}

export default App;
