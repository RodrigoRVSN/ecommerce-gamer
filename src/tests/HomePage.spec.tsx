import React from "react";
import { render } from "@testing-library/react";
import { Home } from "../pages/Home";

describe("<Home />", () => {
  it("should render the home page", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
