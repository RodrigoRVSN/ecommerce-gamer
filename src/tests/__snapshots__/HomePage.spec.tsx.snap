// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render the home page 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <button
      aria-controls="simple-menu"
      aria-haspopup="true"
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-containedSizeLarge MuiButton-sizeLarge"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <img
          alt="filtro"
          src="/assets/arrow-down-icon.svg"
        />
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4"
      style="align-items: center; justify-content: center; text-align: center;"
    />
  </div>
</div>
`;
